source/release-notes/1.2.txt

```diff
@@ -17,14 +17,20 @@ Release Notes for ``mongosync`` 1.2
 Migrating Older Versions of MongoDB
 -------------------------------------
 
-Starting in version 1.2, ``mongosync`` can sync between clusters running 
-some older (pre-6.0) releases of MongoDB Server. This feature requires
-additional preparation and configuration in pre-6.0 releases.
+Starting in version 1.2, ``mongosync`` can sync between clusters running
+some pre-6.0 releases of MongoDB Server for one time migration onto
+Atlas. This feature requires additional preparation and configuration in
+pre-6.0 releases.
 
 If you would like to migrate pre-6.0 versions of MongoDB Server
 using ``mongosync``, please `contact <https://mongodb.com/contact>`__
 your account team to inquire about Professional Services.
 
+.. note::
+
+   ``mongosync`` does not support continuous sync between clusters
+   running pre-6.0 releases of MongoDB Server.  
```

Comment:
I know this existed prior, but consider removing the verbiage "pre-6.0" with this edit. This applies to each occurrence on the page:

Consider:

   ``mongosync`` does not support continuous sync for MongoDB clusters (source or destination) with versions lower than 6.0.

Reply:
Done!